### PR TITLE
distrho: 2017-10-10 -> 2018-01-01

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/audio/distrho/default.nix [hunk 1]
```diff
@@ -2,12 +2,12 @@
 , libxslt, lv2, pkgconfig, premake3, xorg, ladspa-sdk }:
 
 stdenv.mkDerivation rec {
-  name = "distrho-ports-unstable-2017-10-10";
+  name = "distrho-ports-unstable-2018-01-01";
 
   src = fetchgit {
```
Comment (adisbladis) [inline pkgs/applications/audio/distrho/default.nix]: This PR itself is fine but I would appreciate it if we could change this to use `fetchFromGitHub`.

Comment (magnetophon) [reply pkgs/applications/audio/distrho/default.nix]: OK, done.

**Comment by grahamc** [comment]
@GrahamcOfBorg build distrho

**Comment by magnetophon** [comment]
Why is borg building for darwin when the pkg says ``platforms = platforms.linux;``?

**Comment by grahamc** [comment]
It is too dumb to know otherwise :)

**Comment by adisbladis** [comment]
I attempted a fix for the aarch64 build error in https://github.com/NixOS/nixpkgs/pull/34316

**Pull request merged**